src/style.css

```diff
@@ -0,0 +1,81 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.container__block {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: rgb(255, 255, 255);
+  display: flex;
+  justify-content: center;

```

Comment:
Pay attention to your CSS properties order
https://9elements.com/css-rule-order/